src/pages/User-Infor/index.vue

```diff
@@ -0,0 +1,316 @@
+<script setup>
+import { onBeforeMount, reactive } from 'vue';
+import { validateFullName, validateDOB, validataPhoneNumber, validatePassword } from '@/utils/validator';
+import { getUserById, updateUserAddress, updateUserInformation, changeUserPassword, deleteUserById } from '@/api/user.api';
+import { useAuthStore } from '@/stores/auth';
+import { useGeolocation } from '@vueuse/core';
+import router from '@/router';
+import { collapseLight } from 'naive-ui/es/collapse/styles';
+
+const message = useMessage();
+const auth = useAuthStore();
+const loadingBar = useLoadingBar();
+const disabledRef = ref(true);
+const dialog = useDialog()
+const userForm = ref({});
+const user = ref({});
+const account = reactive({
+    username: "",
+    oldPassword: "",
+    password: "",
+    repeatPassword: ""
+})
+const customerId = ref(-1);
+const loadCustomer = async () => {
+    if (!(typeof auth.auth.customerId === 'undefined') || !(typeof auth.auth.customerName === 'undefined')) {
+        customerId.value = auth.auth.customerId;
+        const data = await getUserById(customerId.value);
+        userForm.value = data.data;
+        let timestampDob = new Date(userForm.value.dob).getTime();
+        userForm.value.dob = timestampDob;
+        user.value = {
+            fullName: data.data.fullName,
+            phoneNumber: data.data.phoneNumber,
+            address: data.data.address.addressName
+        };
+        account.username = auth.auth.customerName;
+    }
+}
+onBeforeMount(() => {
+    loadCustomer();
+});
```

Comment:
async await của tôi đâuu
```suggestion
onBeforeMount(async () => {
    await loadCustomer();
});
```